CONTRIBUTING.md

```diff
@@ -203,28 +203,24 @@ on Windows. You will get more stable benchmark results of you end those processe
         * On linux boxes, you have use https://github.com/lpechacek/cpuset.
         * On Windows, you can "Set Processor Affinity" using https://www.thewindowsclub.com/processor-affinity-windows
         * On Mac, you can try to use their dedicated affinity API https://developer.apple.com/library/archive/releasenotes/Performance/RN-AffinityAPI/#//apple_ref/doc/uid/TP40006635-CH1-DontLinkElementID_2
-3. To benchmark, you will likely end up writing a seperate c/c++ program that will link libzstd.
+3. To benchmark, you will likely end up writing a separate c/c++ program that will link libzstd.
 Dynamically linking your library will introduce some added variation (not a large amount but
-definitly some). Statically linking libzstd will be more stable. Static libraries should
+definitely some). Statically linking libzstd will be more stable. Static libraries should
 be enabled by default when building zstd.
-
-Those are the low hanging fruit. You can also try the following more advanced techniques
-if you need to measure smaller changes:
-
-1. Use a profiler with a good high resolution timer. See the section below on profiling for
+4. Use a profiler with a good high resolution timer. See the section below on profiling for
 details on this.
-2. Disable frequency scaling, turbo boost and address space randomization (this will vary by OS)
-3. Try to avoid storage. On some systems you can use tmpfs. Putting the program, inputs and outputs on
+5. Disable frequency scaling, turbo boost and address space randomization (this will vary by OS)
+6. Try to avoid storage. On some systems you can use tmpfs. Putting the program, inputs and outputs on
 tmpfs avoids touching a real storage system, which can have a pretty big variability.
 
 Also check our LLVM's guide on benchmarking here: https://llvm.org/docs/Benchmarking.html
 
-### zstdbench
+### Becnhzstd
```

Comment:
nit: `benchzstd`
or better yet : `zstd` benchmark
(not sure if it's important for this paragraph to be titled with the name of a source file)